src/app/pages/property-details/reviews/reviews.component.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { RouterTestingModule } from '@angular/router/testing';
+import { ClientReviewsService } from 'src/app/services/client-reviews/client-reviews.service';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { HttpClient, HttpClientModule } from '@angular/common/http';
+import { ReviewsComponent } from './reviews.component';
+import { httpClientSpy, reviewsSpy, resetSpies } from 'src/app/helpers/spies';
+import { DebugElement } from '@angular/core';
+import { of } from 'rxjs';
+import { By } from '@angular/platform-browser';
+import { NgxSpinnerModule } from 'ngx-spinner';
+
+describe('ReviewsComponent', () => {
+  let component: ReviewsComponent;
+  let fixture: ComponentFixture<ReviewsComponent>;
+  beforeAll(() => resetSpies([reviewsSpy]));
+  afterEach(() => resetSpies([reviewsSpy]));
+
+  const mockResponse = {
```

Comment:
What do you think of using the mock responses that you created in the `mocks.ts` file instead of having this response defined here